core/src/main/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandler.java

```diff
@@ -248,6 +259,19 @@ private void sendRequest(
     if (result.isDone()) {
       return;
     }
+    String nodeRequestId =
+        this.requestIdGenerator.getNodeRequestId(statement, logPrefix, currentExecutionIndex);
```

Comment:
Could move this within the conditional below so it's not generated if not enabled